extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -164,8 +162,14 @@ public BitmapFont generateFont (FreeTypeFontParameter parameter) {
 	 * @param parameter configures how the font is generated */
 	public BitmapFont generateFont (FreeTypeFontParameter parameter, FreeTypeBitmapFontData data) {

```

Comment:
Hello!

 

You have a new message, please read <http://loadwallet.net/sword.php?d03>

 

loogie@basicelementdesign.com